the-collab-lab/tcl-62-smart-shopping-list
# Creates a header with a logo

## Description

This code updates the title and navigation into a single header at the top of the page. It adds a logo, a background, and updates the navigation tabs to match the color scheme.

## Related Issue

Closes #28 & Closes #29 

## Acceptance Criteria

- [x] Ensure file format is compatible (eg. SVG)
- [x] Ensure background is transparent
- [x] Ensure file size is large enough so it's not pixelated when enlarged on screen
- [x] Header has logo on left side and nav items are on right side. 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|    | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |
| ✓   | :sparkling_heart: Styling     |

## Updates

### Before

<img width="1124" alt="Screenshot 2023-10-02 at 6 13 00 PM" src="https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/115492619/55f0bb88-3aff-47bf-aab6-0d945a4bea21">

### After

<img width="1121" alt="Screenshot 2023-10-02 at 6 12 08 PM" src="https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/115492619/afc22a96-429a-44c6-903a-9bb16b9584fe">

## Testing Steps / QA Criteria

1. Switch to the `logo-header` branch on your local computer
2. Run `npm i` to install the newest dependencies
4. In your web browser view the page and ensure the custom header pictured above is loading.
5. Click the navigation tabs to make sure the work.
6. Navigate to the 'Add Item' page. Click the logo and verify it navigates you back to the home view (which should be the list page).
